### DISCUSSION TITLE
alas打开后闪退，使用console打开后显示错误

### DISCUSSION THREAD

**Discussion by MoYuNEKO** [LmeSzinc/AzurLaneAutoScript#1990] [Q&A] [answered]
昨天还用得好好的，第二天重装了一下系统就用不了了，本来打开alse.exe能过直接运行，现在alas打开后闪退，在使用console打开后输入python gui.py后报错，内容为：
ERROR:    [Errno 10048] error while attempting to bind on address ('0.0.0.0', 22267): 通常每个套接字地址(协议/网络地址/ 端口)只允许使用一次。
![aa284a5a249963424d3f73c6abe1b28](https://user-images.githubusercontent.com/120086511/206908882-7f2c6dc0-bb24-4213-be2b-8b6d4a0f68b5.png)

**Comment by MoYuNEKO** [comment]
请求帮助

**Comment by LmeSzinc** [comment, marked as answer]
已经有一个 Alas 运行在 22267 端口了，不能同时运行两个 Alas。

你可以在浏览器访问 `127.0.0.1:22267` 使用已有的 Alas，或者执行 `taskkill /t /f /im python.exe` 和 `taskkill /t /f /im alas.exe` 杀掉已有的 Alas，来启动新的。（假设你电脑上没有运行其他 python 程序

**Reply by MoYuNEKO** [reply]
将taskkill /t /f /im python.exe在指令框内打出后重复操作后是有效的答案，感谢帮助，已经在浏览器内的alas成功运行，但是win11的alsa启动时闪退的问题就没办法解决吗？

**Reply by LmeSzinc** [reply]
alas 内置的 electron 已经很久没更新了，在 win11 22H2 以上会闪退，有空再更新然后发下一版的启动器

**Reply by MoYuNEKO** [reply]
可能是我在重装系统的时候顺便将win11更新了，最后再次感谢帮助。
